### PR TITLE
Variables: Add support for aliasIDs to datasource variables (Fixes issue with Postgres datasource variables)

### PR DESCRIPTION
**Pull request by torkelo**
There are still a few places where aliasIDs are not taken into account. I expect some more in community/customer plugins. 

Scenes DataSourceVariable did not have this problem as it's using DataSourceSrv.getList with a pluginId filter.

### CODE REVIEW

File: public/app/features/dashboard-scene/utils/getNextPanelId.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import { SceneGridItem, SceneGridLayout, SceneGridRow, SceneObject, VizPanel } from '@grafana/scenes';
+
+import { DashboardScene } from '../scene/DashboardScene';
+import { PanelRepeaterGridItem } from '../scene/PanelRepeaterGridItem';
+
+import { getPanelIdForVizPanel } from './utils';
+
+export class NextPanelIdGenerator {
```
Comment (dprokop) [inline public/app/features/dashboard-scene/utils/getNextPanelId.ts:8]: @torkelo  - Not related to this PR I think?

Comment (torkelo) [reply public/app/features/dashboard-scene/utils/getNextPanelId.ts:8]: @dprokop oops, no that was a test from yesterday :)

**Review by dprokop — APPROVED** [review]
LGTM

**Review by andresmgot — APPROVED** [review]
Tested that this fixes the issue, thanks!

**Pull request merged**